hashview/__init__.py

```diff
@@ -1,9 +1,140 @@
+    return __version__
+
+
+def do_gui_setup_if_needed():
+    from flask import current_app
+    logger = current_app.logger
+
+    from urllib.parse import urlparse
+    static_path = url_for('static', filename='')
+    parsed_url  = urlparse(request.url)
+
+    if parsed_url.path.startswith(static_path):
+        # allow static files through
+        return
+
+    from hashview.models import db
+
+    from hashview.setup import admin_pass_needs_changed
+    from hashview.users.routes import bcrypt
+    if not admin_pass_needs_changed(db, bcrypt):
+        logger.info('Admin password needs changed.')
+
+    else:
+        logger.info('Admin password needs changed.')
+        if (url_for('setup.admin_pass_get') != parsed_url.path):
+            return redirect(url_for('setup.admin_pass_get'))
+        else:
+            return
+
+    from hashview.setup import settings_needs_added
+    if not settings_needs_added(db):
+        logger.info('Settings does not need created.')
+
+    else:
+        logger.info('Settings needs created.')
+        if (url_for('setup.settings_get') != parsed_url.path):
+            return redirect(url_for('setup.settings_get'))
+        else:
+            return
+
+
+def setup_defaults_if_needed():
+    from flask import current_app
+    logger = current_app.logger
+    logger.info('Setting up defaults on before first request.')
+
+    from hashview.models import db
+
+    try:
+        logger.info('Upgrading Database if needed Progressing.')
+        import alembic.command
+        migrate_ext = config = current_app.extensions['migrate']
+        config = migrate_ext.migrate.get_config(migrate_ext.directory)
+        # set configure_logger so that migrations/env.py doesn't override the logging setup
+        config.attributes['configure_logger'] = False
+        alembic.command.upgrade(config, 'head')
+        logger.info('Upgrading Database if needed is Complete.')
+    except:
+        logger.exception('Upgrading Database failed.')
+
+    try:
+        from hashview.scheduler import scheduler
+        from hashview.scheduler import data_retention_cleanup
+        logger.info('Clearing Scheduled Jobs.')
+        scheduler.delete_all_jobs()
+        logger.info('Adding Default Scheduled Jobs Progressing.')
+        scheduler.add_job(id='DATA_RETENTION', func=partial(data_retention_cleanup, current_app), trigger='cron', hour='*')
+        logger.info('Adding Default Scheduled Jobs is Complete.')
+    except:
+        logger.exception('Adding Default Scheduled Jobs failed.')
+
+    try:
+        from hashview.users.routes import bcrypt
+        from hashview.setup import add_admin_user
+        from hashview.setup import admin_user_needs_added
+        if admin_user_needs_added(db):
+            logger.info('Adding Admin User.')
+            add_admin_user(db, bcrypt)
+    except:
+        logger.exception('Adding Admin User failed.')
+
+    try:
```

Comment:
We should swap the order in which the default wordlists are added. Starting with the dynamic wordlist followed by the static wordlist. 

This way, the dynamic wordlist, in the db ends up with id=1.